user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -390,6 +395,41 @@ public void setAuthentication(KafkaUserAuthentication authentication) {
         this.authentication = authentication;
     }
 
+    /**
+     * Sets the quotas to the user.
+     *
+     * @param quotas KafkaUserQuotas to be set.
+     */
+    public void setQuotas(KafkaUserQuotas quotas) {
+        this.quotas = quotas;
+    }
+
+    public JsonObject quotasToJson() {
+        JsonObject quotasJson = new JsonObject();
+        if (quotas == null) {
+            return quotasJson;
+        }
+        if (quotas.getProducerByteRate() != null) {
+            quotasJson.put("producer_byte_rate", quotas.getProducerByteRate().toString());
+        }
+        if (quotas.getConsumerByteRate() != null) {
+            quotasJson.put("consumer_byte_rate", quotas.getConsumerByteRate().toString());
+        }
+        if (quotas.getRequestPercentage() != null) {
+            quotasJson.put("request_percentage", quotas.getRequestPercentage().toString());
+        }
+        return quotasJson;
+    }
```

Comment:
Can you move this to below the getter (all else being equal, it's much nicer if getters and setters are next to each other)